privacyidea/lib/importotp.py

```diff
@@ -461,9 +464,11 @@ def parsePSKCdata(xml_data,
     """
 
     tokens = {}
-    #xml = BeautifulSoup(xml_data, "lxml")
     xml = strip_prefix_from_soup(BeautifulSoup(xml_data, "lxml"))
 
+    if not xml.keycontainer:
+        log.warning('No KeyContainer found in PSKC data. Could not import any tokens.')
```

Comment:
Shouldn't we raise an ``ImportException`` here, so that the error gets propagated to the webui.
Otherwise the importing admin will simply wonder, why zero tokens have been imported.
Something like this might help.

     raise ImportException('This does not seem to be a valid PSKC file. The log level tag "KeyContainer" is missing. No tokens have been imported.')

Please also add a test for this.

Reply:
will do.

Reply:
Done in 204f9f9